File: samples/Go/strip-a-set-of-characters-from-a-string.go
package main

import (
    "fmt"
    "strings"
)

func stripchars(str, chr string) string {
    return strings.Map(func(r rune) rune {
        if strings.IndexRune(chr, r) < 0 {
            return r
        }
        return -1
    }, str)
}

func main() {
    fmt.Println(stripchars("She was a soul stripper. She took my heart!",
        "aei"))
}

File: samples/Lua/strip-a-set-of-characters-from-a-string.lua
function stripchars( str, chr )
    local s = ""
    for g in str:gmatch( "[^"..chr.."]" ) do
 	s = s .. g
    end
    return s
end

print( stripchars( "She was a soul stripper. She took my heart!", "aei" ) )
